### DISCUSSION TITLE
Is nextjs compatible with classnames lib?

### DISCUSSION THREAD

**Discussion by ayrtongomes** [vercel/next.js#14569] [Help] [answered]
I started using scss modules style and I think  I'll need to use the classnames lib in some cases, if is compatible, Do i need to be careful with something?

**Comment by rafaelalmeidatk** [comment, marked as answer]
Yeah it is compatible, there isn't a reason for it to break with Next

**Reply by ayrtongomes** [reply]
Thanks!
